playbooks/clouds.yaml.j2

```diff
@@ -0,0 +1,54 @@
+clouds:
+  gx-scs:
+    auth:
+      auth_url: https://api.gx-scs.sovereignit.cloud:5000
+      application_credential_id: "{{ clouds_conf.gx_scs_ac_id }}"
+      application_credential_secret: {{ clouds_conf.gx_scs_ac_secret }}"
+    region_name: "RegionOne"
+    interface: "public"
+    identity_api_version: 3
+    auth_type: "v3applicationcredential"
+  pco-prod1:
+    region_name: "prod1"
+    interface: "public"
+    identity_api_version: 3
+    auth:
+      auth_url: https://prod1.api.pco.get-cloud.io:5000
+      application_credential_id: "{{ clouds_conf.pco_prod1_ac_id }}"
+      application_credential_secret: {{ clouds_conf.pco_prod1_ac_secret }}"
+    auth_type: "v3applicationcredential"
+  pco-prod2:
+    region_name: "prod2"
+    interface: "public"
+    identity_api_version: 3
+    auth:
+      auth_url: https://prod2.api.pco.get-cloud.io:5000
+      application_credential_id: "{{ clouds_conf.pco_prod2_ac_id }}"
+      application_credential_secret: {{ clouds_conf.pco_prod2_ac_secret }}"
+    auth_type: "v3applicationcredential"
+  pco-prod3:
+    region_name: "prod3"
+    interface: "public"
+    identity_api_version: 3
+    auth:
+      auth_url: https://prod3.api.pco.get-cloud.io:5000
+      application_credential_id: "{{ clouds_conf.pco_prod3_ac_id }}"
+      application_credential_secret: {{ clouds_conf.pco_prod3_ac_secret }}"
+    auth_type: "v3applicationcredential"
+  regio-a:
+    region_name: "RegionA"
+    interface: "public"
+    identity_api_version: 3
+    auth:
+      auth_url: https://keystone.services.a.regiocloud.tech/v3/
+      application_credential_id: "{{ clouds_conf.regio_a_ac_id }}"
+      application_credential_secret: {{ clouds_conf.regio_a_ac_secret }}"
+    auth_type: "v3applicationcredential"
+  wavestack:
+    interface: "public"
+    identity_api_version: 3
+    auth:
+      auth_url: https://api.wavestack.de:5000
+      application_credential_id: "{{ clouds_conf.wavestack_ac_id }}"
+      application_credential_secret: {{ clouds_conf.wavestack_ac_secret }}"
+    auth_type: "v3applicationcredential"
```

Comment:
```suggestion
clouds:
  gx-scs:
    auth:
      auth_url: https://api.gx-scs.sovereignit.cloud:5000
      application_credential_id: "{{ clouds_conf.gx_scs_ac_id }}"
      application_credential_secret: "{{ clouds_conf.gx_scs_ac_secret }}"
    region_name: "RegionOne"
    interface: "public"
    identity_api_version: 3
    auth_type: "v3applicationcredential"
  pco-prod1:
    region_name: "prod1"
    interface: "public"
    identity_api_version: 3
    auth:
      auth_url: https://prod1.api.pco.get-cloud.io:5000
      application_credential_id: "{{ clouds_conf.pco_prod1_ac_id }}"
      application_credential_secret: "{{ clouds_conf.pco_prod1_ac_secret }}"
    auth_type: "v3applicationcredential"
  pco-prod2:
    region_name: "prod2"
    interface: "public"
    identity_api_version: 3
    auth:
      auth_url: https://prod2.api.pco.get-cloud.io:5000
      application_credential_id: "{{ clouds_conf.pco_prod2_ac_id }}"
      application_credential_secret: "{{ clouds_conf.pco_prod2_ac_secret }}"
    auth_type: "v3applicationcredential"
  pco-prod3:
    region_name: "prod3"
    interface: "public"
    identity_api_version: 3
    auth:
      auth_url: https://prod3.api.pco.get-cloud.io:5000
      application_credential_id: "{{ clouds_conf.pco_prod3_ac_id }}"
      application_credential_secret: "{{ clouds_conf.pco_prod3_ac_secret }}"
    auth_type: "v3applicationcredential"
  regio-a:
    region_name: "RegionA"
    interface: "public"
    identity_api_version: 3
    auth:
      auth_url: https://keystone.services.a.regiocloud.tech/v3/
      application_credential_id: "{{ clouds_conf.regio_a_ac_id }}"
      application_credential_secret: "{{ clouds_conf.regio_a_ac_secret }}"
    auth_type: "v3applicationcredential"
  wavestack:
    interface: "public"
    identity_api_version: 3
    auth:
      auth_url: https://api.wavestack.de:5000
      application_credential_id: "{{ clouds_conf.wavestack_ac_id }}"
      application_credential_secret: "{{ clouds_conf.wavestack_ac_secret }}"
    auth_type: "v3applicationcredential"
```

Reply:
@chess-knight Thanks for spotting this! I will fix it, but I can't apply your suggestion directly, because GitHub will not let me choose the correct e-mail address, and then the commit will fail the DCO test. So I will do it manually, just as a heads up.